catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/platform/transformer/transformers/miro/MiroLicenses.scala

```diff
@@ -43,10 +45,10 @@ trait MiroLicenses {
           // so they don't end up on the DLQ.
           case "Do not use" =>
             throw new ShouldNotTransformException(
-              "Usage restriction is 'Do not use'")
+              s"Image $miroId has usage restriction 'Do not use'")
           case "Image withdrawn, see notes" =>
             throw new ShouldNotTransformException(
-              "Usage restriction is 'Image withdrawn'")
+              s"Image $miroId has usage restriction 'Image withdrawn'")
         }
```

Comment:
should we handle and test for the default case explicitly ('_'), rather than allow a MatchError?  I guess that goes to the DLQ so may be what is required.